examples/globe_geojson_to3D.html

```diff
@@ -62,13 +62,15 @@
             ariege.convert = itowns.Feature2Mesh.convert({
                 color: color,
                 extrude: extrude });
+            ariege.projection = view.tileLayer.extent.crs();
```

Comment:
it is not up to the user to specify the target projection. it must be `view.referenceCrs`?

Reply:
Ok I'm looking into changing that

Reply:
Removed, because unnecessary